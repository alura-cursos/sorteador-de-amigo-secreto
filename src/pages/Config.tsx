import Form from "../components/Form"
import Participants from "../components/Participants"
import Play from "../components/Play"

const Config = () => {
  return (
    <>
      <Form />
      <Participants />
      <Play />
    </>
  )
}

export default Config