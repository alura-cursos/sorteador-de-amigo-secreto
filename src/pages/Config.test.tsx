import React from 'react'

import { render } from '../utils/test/customRenderer'

import Config from './Config'

const mockNavigate = jest.fn()

jest.mock('react-router-dom', () => {
  return {
    useNavigate: () => mockNavigate
  }
})

describe('<Config />', () => {
  test('should be rendered', () => {
    const {container} = render(<Config />)

    expect(container).toMatchSnapshot()
  })
})