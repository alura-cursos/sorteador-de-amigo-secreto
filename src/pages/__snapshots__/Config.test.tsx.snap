// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Config /> should be rendered 1`] = `
<div>
  <form>
    <input
      placeholder="Insira os nomes dos participantes"
      type="text"
      value=""
    />
    <button
      disabled=""
      type="submit"
    >
      Adicionar
    </button>
    
  </form>
  <ul />
  <button
    disabled=""
  >
    Iniciar Brincadeira
  </button>
</div>
`;
